### PR TITLE
uptake latest ssi sdk and add applicant to credential application

### PR DESCRIPTION
**Pull request by decentralgabe**
- uptake latest ssi sdk
- add applicant id to credential response building

### CODE REVIEW

File: pkg/authorizationserver/oauth2_auth.go [hunk 1]
```diff
@@ -45,8 +46,8 @@ func (s AuthService) AuthEndpoint(ctx context.Context, rw http.ResponseWriter, r
 		return nil
 	}
 
-	// If the Credential Issuer metadata contains an authorization_server parameter, the authorization detail's
-	// locations common data field MUST be set to the Credential Issuer Identifier value
+	// If the Credential IssuerDID metadata contains an authorization_server parameter, the authorization detail's
+	// locations common data field MUST be set to the Credential IssuerDID Identifier value
```
Comment (andresuribe87) [inline pkg/authorizationserver/oauth2_auth.go:50]: Probably didn't wnat?

File: pkg/server/router/credential_test.go [hunk 2]
```diff
@@ -548,7 +548,7 @@ func TestCredentialRouter(t *testing.T) {
 
 	})
 
-	t.Run("Create Multiple Suspendable Credential Different Issuer SchemaID StatusPurpose Triples", func(tt *testing.T) {
+	t.Run("Create Multiple Suspendable Credential Different IssuerDID SchemaID StatusPurpose Triples", func(tt *testing.T) {
```
Comment (andresuribe87) [inline pkg/server/router/credential_test.go:551]: Was this intentional?

File: pkg/service/manifest/storage/storage.go [hunk 3]
```diff
@@ -26,7 +26,7 @@ const (
 
 type StoredManifest struct {
 	ID          string                      `json:"id"`
-	Issuer      string                      `json:"issuer"`
+	IssuerDID   string                      `json:"issuerDid"`
```
Comment (andresuribe87) [inline pkg/service/manifest/storage/storage.go:29]: Was this intentional?

**Review by andresuribe87 — APPROVED** [review]

File: pkg/authorizationserver/oauth2_auth.go (hunk 1 above)
Comment (decentralgabe) [reply pkg/authorizationserver/oauth2_auth.go:50]: oops

File: pkg/service/manifest/storage/storage.go (hunk 3 above)
Comment (decentralgabe) [reply pkg/service/manifest/storage/storage.go:29]: this one was to be consistent with the others

File: pkg/server/router/credential_test.go (hunk 2 above)
Comment (decentralgabe) [reply pkg/server/router/credential_test.go:551]: this one yes

**Pull request merged**